src/Providers/UrbanAirshipV3.php

```diff
@@ -237,6 +238,14 @@ public function validateBeforePush()
             throw new MissingArgumentException('You have to setMessage() before sending push');
         }
 
+        if (!empty($this->getNamedUsers()) AND !empty($this->getAliases())) {
```

Comment:
I understood that it was sending to both while wns is not working. Maybe i'm wrong?

Reply:
NamedUsers is not supported on WNS, but is supported on Android and iOS. This check ensures that we wither use Named Users or Alias but not both in the same call. The next check ensures we never used Named Users when sending to WNS.
So the solution, when we have a mixture of both, is to send to all platforms where we have aliases and to send to all but WNS when we have NamedUsers.